open-craft/sprints
# [BB-1497] Allow merging social account into an existing one

This adds an option to merge social account into existing one (created with email + password). It also adds an ability to override JWT expiration deltas.
This also fixes `daterange` by making it exclusive at the end (as this is how the events are returned from Google Calendar).

## Testing instructions:
1. Set `.env`.
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Navigate to `frontend` and run `npm i && npm start`.
1. Go to http://localhost:8000/admin/socialaccount/socialapp/add/ and set the social application according to the comment in the ticket.
1. Remove user that has your standard email (via Django admin).
1. Register with email + password.
1. Run `docker-compose -f local.yml logs django` to catch confirmation link (or just confirm your new account via Django admin).
1. Use Google button to log in.
1. Check that Samuel on the first board has 14 under 'vacations' instead of 21.
